store/src/main/java/org/apache/rocketmq/store/DefaultMessageStore.java

```diff
@@ -168,6 +168,8 @@ public class DefaultMessageStore implements MessageStore {
     private final ConcurrentMap<Integer /* level */, Long/* delay timeMillis */> delayLevelTable =
         new ConcurrentHashMap<Integer, Long>(32);
 
+    private ConcurrentMap<String/* topic@group */, ConcurrentMap<Integer, Long>> offsetTable;
```

Comment:
There is no need to add a consumer offset map here. Perhaps we can modify the consummeQueueStore of the broker module through the register hook.